WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverCoordinator.swift

```diff
@@ -56,7 +55,9 @@ final class CompliancePopoverCoordinator: CompliancePopoverCoordinatorProtocol {
         let complianceViewController = CompliancePopoverViewController(viewModel: complianceViewModel)
         let bottomSheetViewController = BottomSheetViewController(childViewController: complianceViewController, customHeaderSpacing: 0)
 
-        bottomSheetViewController.show(from: self.viewController)
+        bottomSheetViewController.show(from: viewController)
+
+        self.presentingViewController = viewController
```

Comment:
❓ What is the reason we accept `presentingViewController` in the init **AND** `viewController` in the `presentIfNeeded` function? This way we can init it with XVC and then call the present on YVC resulting in updating XVC to YVC without giving any hints to the call-site.

Reply:
I moved the passing of viewController from `init` to `presentIfNeeded` because if we pass it to `init`, `weak var` will be immediately deallocated.

When I pass it to `presentIfNeeded`, I only assign it to `weak var` once I present it. The navigation stack now holds viewController strongly.